orttraining/orttraining/python/training/ortmodule/__init__.py

```diff
@@ -41,7 +41,7 @@
 elif os.path.exists(os.path.join(TORCH_CPP_BUILD_DIR,'lock')):
     print("WARNING: ORTModule detected PyTorch's CPP extension lock file during initialization, "
           "which can cause the script to stop responding. "
-          f"Delete {os.path.join(TORCH_CPP_BUILD_DIR,'lock')} if a hang occurs.")
+          f"Delete {os.path.join(TORCH_CPP_BUILD_DIR,'lock')} if the script stop responding and not exiting.")
```

Comment:
This us unrelated to this PR and it is being completely reworked at https://github.com/microsoft/onnxruntime/pull/8027/files. Try not changing this code to prevent conflicts between both PRs

Reply:
Sure, thanks for the heads up.

Reply:
Fixed, please check.